index.html

```diff
@@ -1670,7 +1671,19 @@ <h6 id="async-action-response">Asynchronous Action Response</h6>
               "timeRequested": "2021-11-10T11:43:19.135Z"
             }
           </pre>
-        </section>
+          <div class="rfc2119-assertion" id="http-basic-profile-protocol-binding-invokeaction-22">
+          <p>
+            In resource constrained environments, if the buffer becomes full then a Web Thing SHOULD
+            remove the oldest Action result ActionStatus from its buffer to make room for a newer one.
+          </p>
+          </div>
+          <div class="rfc2119-assertion" id="http-basic-profile-protocol-binding-invokeaction-23">
+          <p>
+            A Web Thing MUST return a <code>503</code> error response if the invocation cannot be enqueued,
+            e.g. the internal pending queue is full.
```

Comment:
This might need to be more specific, because there are other reasons an action invocation may not be accepted which might better fit a different response code like `400`, `401`, `403`, `404` or `500`. 

```suggestion
            A Web Thing SHOULD return a <code>503</code> error response if the invocation cannot be accepted because the action is unavailable,
            e.g. because the Thing is overloaded.
```

Reply:
Agreed, thank you for the review.